api/desecapi/models.py

```diff
@@ -216,80 +216,87 @@ def write_rrsets(self, datas):
 
     @transaction.atomic
     def _write_rrsets(self, rrsets):
+        # Base queryset for all RRset of the current domain
+        rrset_qs = RRset.objects.filter(domain=self)
+
+        # Set to check RRset uniqueness
+        rrsets_seen = set()
+
+        # To-do list for non-empty RRsets, indexed by (subname, type)
+        rrsets_meaty_todo = {}
+
+        # RRsets to send to pdns
```

Comment:
better: dictionary of sets of RRs to send to pdns, indexed by their rrsets objects

Reply:
done